gobblin-runtime/src/main/java/org/apache/gobblin/runtime/Task.java

```diff
@@ -95,6 +78,21 @@
 import org.apache.gobblin.writer.WatermarkAwareWriter;
 import org.apache.gobblin.writer.WatermarkManager;
 import org.apache.gobblin.writer.WatermarkStorage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+import com.google.common.base.Optional;
```

Comment:
Is the ordering of imports correct? I thought org.apache.gobblin should come after com.google.*.